src/harness/fake_sas.py

```diff
@@ -409,16 +409,40 @@ def do_GET(self):
     self.wfile.write(json.dumps(response))
 
 
-def RunFakeServer(version, is_ecc):
+def RunFakeServer(version, is_ecc, ca_cert, verify_crl):
   FakeSasHandler.SetVersion(version)
   if is_ecc:
     assert ssl.HAS_ECDH
+  if ca_cert is not None:
```

Comment:
Instead of allowing `ca_cert` to be `None`, just compute the correct path before calling this function.

We should only need to read `os.path.join('certs', ...)` or `CA_CERT` one time.

Reply:
We have   passed the cert_path and removed none check